test-prow-e2e.sh

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env bash
+
+set -eExuo pipefail
+
+NAMESPACE=""
+DF_CONSOLE_IMAGE=""
```

Comment:
we have finalized the name of the namespace, right ?
same with `DF_CONSOLE_IMAGE` it was received as an argument in ODF, is something different here ?